apps/client/src/App.css

```diff
@@ -210,4 +210,106 @@ footer
   display: flex;
   gap: 5px;
   align-items: center;
-}
\ No newline at end of file
+}
+
+
+
+.gallery {
+  display: flex; 
+  justify-content: center; 
+}
+
+.row {
+  display: flex; 
+  gap: 100px; 
+}
+
+
+.admin-coupon-list {
+  font-family: Arial, sans-serif;
+}
+
+
+.popup {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+
+.popup-inner {
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 5px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.3);
+}
+
+
+.coupon-list {
+  list-style: none;
+  padding: 0;
+}
+
+
+.coupon-item {
+  margin-bottom: 20px;
+  border: 1px solid #ddd;
+  padding: 10px;
+  border-radius: 5px;
+}
+
+
+.coupon-item button {
+  margin-left: 10px;
+  padding: 5px 10px;
+  border: none;
+  background-color: #007bff;
+  color: #fff;
+  border-radius: 3px;
+  cursor: pointer;
+}
```

Comment:
@noagedo 
I see you design all the button almost the same 
consider create button component with one style the we can use all over the app